contracts/satoshi-bridge/src/psbt.rs

```diff
@@ -185,7 +185,7 @@ impl Contract {
             }
         });
         require!(
-            actual_received_amounts.len() == 1,
+            actual_received_amounts.len() <= 1,
             "only one user output is allowed."
         );
         let actual_received_amount = actual_received_amounts[0];
```

Comment:
Next come the checks of the amount the user will receive. In shielded transactions we don’t know this amount and don’t know how much was spent on gas.